web/src/narrow_error.ts

```diff
@@ -5,14 +5,14 @@ type QueryWord = {
     is_stop_word: boolean;
 };
 
-type SearchData = {
-    query_word: QueryWord[];
+export type SearchData = {
+    query_words: QueryWord[];
```

Comment:
this was just wrong before

Reply:
Typically the way to handle this would be to just have a prep commit fixing this bug rather than a GitHub comment.